string type
shape_msgs/Mesh geometry
